lib/fetching/content-services/dummy-content-service.js

```diff
@@ -52,4 +54,8 @@ DummyContentService.prototype._emitNext = function() {
     setTimeout(function() { self._emitNext(); }, 1);
 };
 
+DummyContentService.prototype._reloadSettings = function() {
```

Comment:
This is not a private method since it's called from outside. So it shouldn't start with _